### DISCUSSION TITLE
Netdata v1.43.0 Release Meetup - Groundbreaking: `systemd-journal` logs release!

### DISCUSSION THREAD

**Discussion by hugovalente-pm** [netdata/netdata#16232] [Announcements]
<image src="https://user-images.githubusercontent.com/2662304/275458055-14257b47-f374-4a58-bff9-d3b217a9421b.gif" width="65%" height="65%"/>

Join **Costas Tsaousis**, Founder & CEO, and the **Netdata Product team**, on the 18th of October at 15:30 UTC
for the **Netdata Release Meetup**.

Together we’ll cover:
* Release Highlights
* Netdata brings in system journals!
* Acknowledgments
* Q&A with the community

Join us at:
* [YouTube](https://www.youtube.com/watch?v=YRTDQ1CsE3Y)
* [LinkedIn](https://www.linkedin.com/events/7117060672575684608/)

Check here for the full version of release notes 👉 https://github.com/netdata/netdata/releases/tag/v1.43.0

**Comment by pomazanbohdan** [comment]
How to add local node logs to docker-composer?
